### DISCUSSION TITLE
Suggesting 'drawio' for producing diagrams and other graphics

### DISCUSSION THREAD

**Discussion by imagingbook** [Digital-Media/HagenbergThesis#162] [Ideas]
Just came across `drawio`, which provides a simple and versatile environment for drawing diagrams and other structured graphics. Among other features it offers clean vector export to PDF (including "cropped" and "selection only") and built-in LaTeX (math) editing rendered with MathJAX. I think it is superior to all other methods we had suggested so far and should be addressed in the Hagenberg user docs.

`drawio` is available as an **online editor** (supporting local storage) as well as a **desktop client**.

Links:

- https://www.drawio.com/
- https://www.drawio.com/example-diagrams
- https://www.drawio.com/doc/faq/math-typesetting
- Online editor: https://app.diagrams.net/
- Desktop client: https://github.com/jgraph/drawio-desktop/releases

**Comment by hochleitner** [comment]
Looks good. Are you suggesting a wiki article or replacing parts of the tutorial document with it?

**Reply by imagingbook** [reply]
I think it should fit well into tutorial section 4.3.5, perhaps by adding a new paragraph on producing diagrams with 1-2 examples. More details could go into a wiki article. Some alignment with section 4.3.6 may be necessary...
What do you think?

**Reply by hochleitner** [reply]
Sounds reasonable. I think, if it's really the simplest way to create diagrams for LaTeX, we should include it.
